.github/ISSUE_TEMPLATE/questions.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Question
+about: Have a question about the project? Ask it here!
+title: Concisely state your question
```

Comment:
The *concise* trips me off a bit, as it might give a discouraging feeling towards elaborating on a topic. Could we just name it "Title of your question"?

Reply:
I personally think titles should be fairly concise; elaborate detail can go in the description. We aren't asking for the description to be concise.

Regardless, I've made it a bit friendlier in the latest change.